editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -140,6 +141,22 @@ export const editorStore = (
     );
   },
 
+  copyFlow: async (flowId, teamId) => {
+    const token = getCookie("jwt");
+
+    return axios.post(
+      `${process.env.REACT_APP_API_URL}/flows/${flowId}/copy`,
```

Comment:
These are important questions ! 

My rational for API here was essentially: 
- `makeUniqueFlow` was already defined in the API helper methods because another endpoint depends on it, I don't want to repeat it in the Editor too (`dataMerged` is an example of another helper that is defined in both places currently, and definitely supports a case I think for an isomorphic node-module for logic methods like these)
- recognizing that we don't have the "move" action just yet, I imagine there may be occassional value in a dev being able to manually hit this function as an endpoint and just generate the unique copied flow data that could then be manually pasted into a new Hasura record under _any_ team (in addition to content editors being able to access from Editor)